source/overview/license-and-subscription.rst

```diff
@@ -109,6 +109,71 @@ Is my license available immediately?
 
 Yes, once your payment is successfully processed your license is immediately available to download.
 
+How will I know when to renew my license?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You will be notified 60 days prior to your license expiry that your license is due for renewal, via a blue banner displayed at the top of your Mattermost window. This banner is only visible to System Admins.
+
+You can select **Please renew** to begin the renewal process. You can also select the **x** to dismiss the notification. The notification is reactivated when your browser refreshes.
+
+How do I renew my license?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can renew your license in Mattermost via the blue renewal reminder banner. Select Renew Now to begin the process. You can also visit https://mattermost.com/renew, and complete the form provided. 
+
+How long does it take to renew a license?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Once you’ve started the renewal process, we will be in contact with you to confirm your order and send you the order form. There may be additional paperwork required. Once we have the signed order form and (if applicable) the necessary paperwork from you, we are able to process the renewal and issue your license key within 24 hours.  
+
+
+What happens to my trial/subscription if I don't renew in time?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+From version 5.24, if you don't renew within the 60-day renewal period, a 10 day grace period is provided. During this period your Mattermost installation runs as normal, with full access to Enterprise features. 
+
+When the grace period expires, your Enterprise version is downgraded to Team Edition. Enterprise features are disabled.
+ 
+What happens when my trial/subscription expires?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you don't renew your license within the 10-day grace period, your Mattermost version is automatically downgraded to Team Edition so you can still access and use Mattermost. However, Enterprise features will no longer be available and if you are currently using them, the functionality will no longer be accessible. 
+
+When you renew, the Enterprise features will become available with the previous configuration (provided no action such as user migration has been taken). 
+
+Which features are affected when my Enterprise license expires?
```

Comment:
I'm wondering if this is part of the previous question above, "What happens when my trial/subscription expires?"

Reply:
It was. They are similar questions but one is focused more on the mechanics of the expiry and one on the features affected. I'm open to suggestions - wondering if maybe the questions are what needs work.

Reply:
True. It could be a subsection of the FAQ entry above it. 0/5 though

Reply:
I like the idea of a subsection. That'll work.

Reply:
Subsection added.